python/app/model_publish_dependency_down.py

```diff
@@ -22,7 +22,7 @@ class SgPublishDependencyDownstreamListingModel(SgEntityListingModel):
 
     # note: no constructor implemented - use base class version
 
-    def load_data(self, sg_location):
+    def load_data(self, sg_location, filters=[]):
```

Comment:
Same comment as above
```suggestion
    def load_data(self, sg_location, filters=None):
```